### PR TITLE
I build a browser extension for json diff and would like to add the reference of it in README.md

### PR DESCRIPTION
**Pull request by gangadharjannu**
Adds a reference to json diff browser extension

Allows running json diff even  offline

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,26 @@ docker build --tag=jdd:v1-alpine --file=Dockerfile.alpine .
 docker run -i --name jdd -p 127.0.0.1:8080:80/tcp jdd:v1-alpine
 ```
 
+## Run JSONDiff as an offline browser extension
+
+[@gangadharjannu](https://www.github.com/gangadharjannu) has build the browser extension which works entirely offline.
```
Comment (zgrossbart) [inline README.md:40]:
```suggestion
[@gangadharjannu](https://www.github.com/gangadharjannu) has built a JSONDiff browser extension which works entirely offline.
```

File: README.md [hunk 2]
```diff
@@ -35,6 +35,26 @@ docker build --tag=jdd:v1-alpine --file=Dockerfile.alpine .
...
+
+You can also install JSONDiff as a browser extension and try to run on your laptop whether you are connected to internet or not.
```
Comment (zgrossbart) [inline README.md:42]:
```suggestion
You can install JSONDiff as a Chrome or Firefox browser extension and try to run on your laptop whether you are connected to internet or not.
```

**Review by zgrossbart — CHANGES_REQUESTED** [review]
Looks great.  Just a few small suggestions.

File: README.md (hunk 1 above)
Comment (gangadharjannu) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (gangadharjannu) [reply README.md]: Updated as per suggestion

**Review by zgrossbart — APPROVED** [review]
Looks great.  I'm going to merge this in.

**Comment by gangadharjannu** [comment]
> Looks great. I'm going to merge this in.

Thank you so much :)

**Pull request merged**